### PR TITLE
chore(build): update release github-action script to generate amd64 and arm64 fro Linux and Darwin system 

### PR DESCRIPTION
**Pull request by fanweixiao**
Developers can get `yomo` binary by: 

```bash
$ curl -sL https://github.com/yomorun/cli/releases/download/v0.1.3/yomo-v0.1.3-`uname -m`-`uname -s`.tar.gz | tar xvfz -
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,14 +15,14 @@ vet:
 build:
 	$(GO) build -o bin/yomo -ldflags "-s -w ${GO_LDFLAGS}" ./yomo/main.go
 
-build-arm:
-	GOARCH=arm64 GOOS=linux $(GO) build -o bin/yomo-${VER}-aarch64-linux -ldflags "-s -w ${GO_LDFLAGS}" ./yomo/main.go
+build-release:
+	GOARCH=arm64 GOOS=darwin $(GO) build -o bin/yomo-${VER}-arm64-Darwin -ldflags "-s -w ${GO_LDFLAGS}" ./yomo/main.go
+	GOARCH=amd64 GOOS=darwin $(GO) build -o bin/yomo-${VER}-x86_64-Darwin -ldflags "-s -w ${GO_LDFLAGS}" ./yomo/main.go
+	GOARCH=arm64 GOOS=linux $(GO) build -o bin/yomo-${VER}-arm64-Linux -ldflags "-s -w ${GO_LDFLAGS}" ./yomo/main.go
+	GOARCH=amd64 GOOS=linux $(GO) build -o bin/yomo-${VER}-x86_64-Linux -ldflags "-s -w ${GO_LDFLAGS}" ./yomo/main.go
```
Comment (venjiang) [inline Makefile:23]: Why is there no Windows?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,21 @@
 # YoMo CLI
+
 Command-line tools for YoMo
 
-## Prerequisites
+## Binary
+
+`curl -sL https://github.com/yomorun/cli/releases/download/v0.1.3/yomo-v0.1.3-`uname -m`-`uname -s`.tar.gz | tar xvfz -`
+
```
Comment (venjiang) [inline README.md:8]: [Bina](https://bina.egoist.sh) What do you think?

**Comment by fanweixiao** [comment]
@venjiang add `bina.json`

**Review by venjiang — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (fanweixiao) [reply Makefile:23]: Will add windows support

File: README.md (hunk 2 above)
Comment (fanweixiao) [reply README.md:8]: added

**Pull request merged**